<?xml version="1.0" encoding="UTF-8"?>
<!--
SPDX-FileCopyrightText: 2025 Qian
SPDX-License-Identifier: AGPL-3.0-only
-->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1400</width>
      <height>1050</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>学生表</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget_student">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>1000</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_secondary">
     <attribute name="title">
      <string>教师表</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_secondary">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_tertiary">
     <attribute name="title">
      <string>课程表</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_tertiary">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget_course">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1400</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>350</y>
      <width>531</width>
      <height>806</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_manage">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::WrapAllRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="edittext_SQL_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="plainText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>SELECT * FROM [tablename] WHERE [expression] ORDER BY column ASC/DESC UPDATE [tablename] SET [column=value] WHERE [column=value]                           INSERT INTO [tablename]  (arg1,arg2,arg3,...) VALUES(arg1,arg2,arg3,...)                  DELETE * FROM [tablename] WHERE [column=value] ORDER BY column ASC/DESC ALTER TABLE [tablename] ADD INDEX [idx_name] [column]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="button_execute">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>160</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_show_origin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>160</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查看原表</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>30</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_show_info">
     <item row="2" column="1">
      <widget class="QLabel" name="label_showExtraInfo1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_showId">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_extraInfo1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_showName">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_extraInfo2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_showExtraInfo2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>欢迎！管理员 张三</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="menuItem_new"/>
    <addaction name="menuItem_open"/>
    <addaction name="menuItem_save"/>
    <addaction name="menuItem_save_as"/>
    <addaction name="separator"/>
    <addaction name="menuItem_export"/>
    <addaction name="menuItem_print"/>
    <addaction name="separator"/>
    <addaction name="menuItem_exit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="menuItem_list_view"/>
    <addaction name="menuItem_tree_view"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="menuItem_help"/>
    <addaction name="menuItem_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuItem_new">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menuItem_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menuItem_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menuItem_save_as">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="menuItem_list_view">
   <property name="text">
    <string>列表视图</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="menuItem_tree_view">
   <property name="text">
    <string>树形视图</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="menuItem_table_view">
   <property name="text">
    <string>表格视图</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="menuItem_help">
   <property name="text">
    <string>查看帮助</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="menuItem_about">
   <property name="text">
    <string>关于Teaching Management App</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="menuItem_export">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="menuItem_print">
   <property name="text">
    <string>打印</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="menuItem_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
